domains/pallets/messenger/src/tests.rs

```diff
@@ -1700,4 +1700,225 @@ mod benchmark_fixtures_gen {
         )
         .unwrap();
     }
+
+    #[test]
```

Comment:
ideal to put them behind benchmarks features so that the fixture is not generated everytime

Reply:
They are inside `mod benchmark_fixtures_gen`, which is behind `runtime-benchmarks`, just like other fixture generator tests.